client/src/components/search-page/Dogs.js

```diff
@@ -4,6 +4,18 @@ import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 import { GiSniffingDog } from "react-icons/gi";
 
+import dogOne from "../../images/dogs/dogOne.png";
+import dogTwo from "../../images/dogs/dogTwo.png";
+import dogThree from "../../images/dogs/dogThree.png";
+import dogFour from "../../images/dogs/dogFour.png";
+import dogFive from "../../images/dogs/dogFive.png";
+
+const fakeFrensArray = [dogOne, dogTwo, dogThree, dogFour, dogFive];
+
+function getRandomInt(max) {
+  return Math.floor(Math.random() * max);
+}
```

Comment:
I have seen this function in another file. Maybe you could refactor this to a util function and export this. So you don't need to write this code every time you need it. You just import it.